include/libdnf/rpm/config_repo.hpp

```diff
@@ -61,20 +51,20 @@ class ConfigRepo : public Config {
     OptionChild<OptionString> & proxy();
     OptionChild<OptionString> & proxy_username();
     OptionChild<OptionString> & proxy_password();
-    OptionChild<OptionEnum<std::string> > & proxy_auth_method();
+    OptionChild<OptionEnum<std::string>> & proxy_auth_method();
     OptionChild<OptionString> & username();
     OptionChild<OptionString> & password();
     OptionChild<OptionStringList> & protected_packages();
     OptionChild<OptionBool> & gpgcheck();
     OptionChild<OptionBool> & repo_gpgcheck();
```

Comment:
Could you consider addressing the gpgcheck RFE with this PR? https://bugzilla.redhat.com/show_bug.cgi?id=1433592#c7

Reply:
Not with this PR. We're definitely planning gpgcheck improvements, but we need to get libdnf in a better shape first. Now it's unusable and DNF doesn't work with it.